addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -290,6 +372,13 @@ function factory(dependencies) {
         isLoading: attr({
             default: false,
         }),
+        /**
+         * Determines whether the cursor is currently over this thread name in
+         * the top bar.
+         */
+        isOverThreadName: attr({
+            default: false,
+        }),
```

Comment:
Cursor/Mouse in name of field looks essential. Omission makes it hard to understand field just by its name.

Reply:
good idea